IncludeToolbox/Formatter/IncludeFormatter.cs

```diff
@@ -213,8 +250,32 @@ private static List<IncludeLineInfo> SortIncludes(IList<IncludeLineInfo> lines,
         /// <returns>Formated text.</returns>
         public static string FormatIncludes(string text, string documentPath, IEnumerable<string> includeDirectories, FormatterOptionsPage settings)
         {
-            string documentDir = Path.GetDirectoryName(documentPath);
+            string documentDir = Utils.GetExactPathName(Path.GetDirectoryName(documentPath));
             string documentName = Path.GetFileNameWithoutExtension(documentPath);
+            string includeRootDirectory = null;
```

Comment:
The following loops up through parent directories looking for the file specified in the settings.